code/modules/admin/topic.dm

```diff
@@ -2425,8 +2425,32 @@
 			to_chat(H, "<span class='specialnoticebold'>Incoming priority transmission from Central Command. Message as follows,</span><span class='specialnotice'> Your ERT request has been denied for the following reasons: [input].</span>")
 		else
 			src.owner.response_team()
+			
+	else if(href_list["GammaReply"])
+		if(!check_rights(R_ADMIN))
+			return
 
+		if(alert(src.owner, "Accept or Deny Gamma Alert request?", "CentComm Response", "Accept", "Deny") == "Deny")
+			var/mob/living/carbon/human/H = locateUID(href_list["GammaReply"])
+			if(!istype(H))
+				to_chat(usr, "<span class='warning'>This can only be used on instances of type /mob/living/carbon/human</span>")
+				return
+			if(H.stat != 0)
```

Comment:
```suggestion
			if(H.stat != CONSCIOUS)
```
Please don't use hardcoded numbers like 0 when defines exist for the thing you are checking.